src/app/pages/StakingPage/Features/DelegationList/index.tsx

```diff
@@ -104,8 +104,11 @@ export const DelegationList = memo((props: Props) => {
       id: 'fee',
       selector: 'fee',
       sortable: true,
-      width: '100px',
-      cell: datum => (datum.validator?.fee !== undefined ? `${datum.validator.fee}%` : 'Unknown'),
+      width: '110px',
+      cell: datum =>
+        datum.validator?.current_rate !== undefined
+          ? `${datum.validator.current_rate.rate * 100}%`
+          : 'Unknown',
```

Comment:
come to think of it, we were :pinching_hand: this close to having it say "Unknown" instead of "undefined%"